packages/cli/src/commands/env/rm.ts

```diff
@@ -0,0 +1,41 @@
+import * as api from '../../rest/api'
+import { Args } from '@oclif/core'
+import { AuthCommand } from '../authCommand'
+
+export default class EnvRm extends AuthCommand {
```

Comment:
@hlenke , shouldn't this command have a confirmation dialog `--force` flag?

Reply:
@nahuelon good catch! I added a --force option to it.